docs/installation.md

```diff
@@ -183,248 +228,204 @@ In short, you would need to
 }
 ```
 
-- If you have issues with duplicate symbols you will need to enable dead code stripping option in your Xcode (Target > Build Settings > search for "Dead code stripping") [see here](https://github.com/firebase/quickstart-ios/issues/487#issuecomment-415313053).
-- If you are using `pod Firebase/Core` with a version set below 5.13 you might want to add `pod 'GoogleAppMeasurement', '~> 5.3.0'` to your podfile
-
-### Android
-
-1. `npm install react-native-camera --save`
-2. Open up `android/app/src/main/java/[...]/MainApplication.java`
-
-- Add `import org.reactnative.camera.RNCameraPackage;` to the imports at the top of the file
-- Add `new RNCameraPackage()` to the list returned by the `getPackages()` method. Add a comma to the previous item if there's already something there.
-
-3. Append the following lines to `android/settings.gradle`:
+<details>
+  <summary>Additional information in case of problems</summary>
 
-   ```gradle
-   include ':react-native-camera'
-   project(':react-native-camera').projectDir = new File(rootProject.projectDir, 	'../node_modules/react-native-camera/android')
-   ```
+  - If you have issues with duplicate symbols you will need to enable dead code stripping option in your Xcode (Target > Build Settings > search for "Dead code stripping") [see here](https://github.com/firebase/quickstart-ios/issues/487#issuecomment-415313053).
+  - If you are using `pod Firebase/Core` with a version set below 5.13 you might want to add `pod 'GoogleAppMeasurement', '~> 5.3.0'` to your podfile
+</details>
 
-4. Insert the following lines in `android/app/build.gradle`:
 
-inside the dependencies block:
+## Android
 
-    ```gradle
-    implementation project(':react-native-camera')
-    ```
+### Modifying build.gradle
 
-inside defaultConfig block insert either:
+Modify the following lines in `android/app/build.gradle`:
 
 ```gradle
 android {
   ...
   defaultConfig {
     ...
-    missingDimensionStrategy 'react-native-camera', 'general' <-- insert this line
+    missingDimensionStrategy 'react-native-camera', 'mlkit' // <--- replace general with mlkit
   }
 }
 ```
 
-or, if using MLKit for text/face/barcode recognition:
+### Setting up Firebase
```

Comment:
For android I actually managed to make barcode recognition working without installing firebase. Not sure why.
I used rn `0.61.4` with `npx react-native init`